packages/utils/src/carequality/cq-monthly-stats/index.ts

```diff
@@ -0,0 +1,52 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { QueryTypes } from "sequelize";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import fs from "fs";
+import { sleep } from "@metriport/shared";
+import { xcpdStats } from "./xcpd-stats";
+import { xcaDQStats } from "./xca-dq-stats";
+import { xcaDRStats } from "./xca-dr-stats";
+import { readOnlyDBPool } from "./shared";
+
+dayjs.extend(duration);
+
+async function main() {
+  const sqlCQDirectory = `SELECT * FROM cq_directory_entry`;
+  const cqDirectory = await readOnlyDBPool.query(sqlCQDirectory, {
+    type: QueryTypes.SELECT,
+  });
+
+  console.log("cqDirectory:", cqDirectory.length);
+
+  const previousMonth = dayjs().subtract(1, "month");
+  const daysInPreviousMonth = previousMonth.daysInMonth();
+  const endOfPreviousMonth = previousMonth.endOf("month").format("YYYY-MM-DD");
+
+  for (let i = 0; i < daysInPreviousMonth; i++) {
```

Comment:
Later we are going to optimize to have all the data be aggregated into one JSON file for each request type but for now were leaving it as seperate folders for each day of the month.